staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd_test.go

```diff
@@ -59,7 +59,7 @@ func newStorage(t *testing.T) (customresource.CustomResourceStorage, *etcdtestin
 		&metav1.DeleteOptions{},
 	)
 
-	typer := apiserver.UnstructuredObjectTyper{
+	typer := apiserver.unstructuredObjectTyper{
```

Comment:
This doesn't compile, does it?

Reply:
No, it doesn't. We shouldn't leak from the apiserver into the registry in the first place though.